website/src/contribute/release-blocking.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: section
+title: "Release blockers"
+permalink: /contribute/release-blockers/
+section_menu: section-menu/contribute.html
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Release blockers
+
+A release blocking Jira is any open Jira that has its `Fix Version` field set
```

Comment:
I believe fix version is used as a project planning tool and people are flexible in changing fix version dates during the first bug scrub that happens in the release cycle.

Does JIRA have a field for planning to target a version without a hard intention to block that release?

Reply:
There is no existing functionality for that. We could have a convention. For example Pablo mentioned that he makes bugs "Blocker" in order to block the release. So a bug that is not a "Blocker" but has Fix Version set could be not release blocking. But I think that is probably confusing and causes more work for the release manager.